test/cppsuite/src/main/thread_worker.cpp

```diff
@@ -282,4 +282,22 @@ thread_worker::running() const
 {
     return (_running);
 }
+
```

Comment:
As discussed, please have an example here that covers the usage of the two functions below.